cmd/xdpderper/xdpderper.go

```diff
@@ -41,8 +42,17 @@ func main() {
 	default:
 		log.Fatal("invalid mode")
 	}
+	deviceName := *flagDevice
+	if deviceName == "" {
+		var err error
+		deviceName, _, err = netutil.DefaultInterfacePortable()
+		if err != nil || deviceName == "" {
+			log.Fatalf("failed to detect default route interface: %v", err)
+		}
```

Comment:
should we log what was autodetected here? or perhaps as part of the existing log message on L68 just always log the device name being attached to?